### PR TITLE
Workspace Context Menu Restyling

### PR DESCRIPTION
**Pull request by OliverEGreen**
### Purpose

This PR consists of entirely style-based changes to the workspace context menu, in line with the latest visual refresh design. 

trX4EIyY6s

Note: Apologies to @QilongTang, I had to run XAML Restyler on `InCanvasSearchControl`, it was getting very hard to read as it was.

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers
@QilongTang 
@Amoursol 

### FYIs
@SHKnudsen

### CODE REVIEW

**Comment by OliverEGreen** [comment]
> According to the gif, looks like the items are not left-aligned like in the pic below. I did not test your changes locally. Screen Shot 2021-10-29 at 11 56 58 AM

@reddyashish Ah, now I can see what you mean. This is as-per the design I was given in Figma:
image

**Comment by Amoursol** [comment]
> > According to the gif, looks like the items are not left-aligned like in the pic below. I did not test your changes locally. Screen Shot 2021-10-29 at 11 56 58 AM
> 
> @reddyashish Ah, now I can see what you mean. This is as-per the design I was given in Figma: image

@Jingyi-Wen thoughts on this? I think aligning would be better as a default state, but want to make sure we're consistent with our other approaches.

**Comment by Jingyi-Wen** [comment]
> > > According to the gif, looks like the items are not left-aligned like in the pic below. I did not test your changes locally. Screen Shot 2021-10-29 at 11 56 58 AM
> > 
> > 
> > @reddyashish Ah, now I can see what you mean. This is as-per the design I was given in Figma: image
> 
> @Jingyi-Wen thoughts on this? I think aligning would be better as a default state, but want to make sure we're consistent with our other approaches.

Just checked with Elizabeth, it should still be the same alignment as current. Mockup will be updated soon

**Comment by OliverEGreen** [comment]
I spoke with UI/UX, we agreed to align the Search textbox with the context menu items. 
@reddyashish @QilongTang 

D7J2SGke4Z

**Comment by OliverEGreen** [comment]
It's probably also worth removing the 16px margin next to the search result icons:

image

**Review by reddyashish — APPROVED** [review]
Looks good to me.

File: src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml [hunk 1]
```diff
@@ -42,93 +38,79 @@
...
+                               Width="20px"
+                               Height="20px"
+                               HorizontalAlignment="Left"
                                VerticalAlignment="Center"
-                               HorizontalAlignment="Left">
+                               DockPanel.Dock="Right">
                             <Image.Style>
                                 <Style TargetType="{x:Type Image}">
-                                    <Setter Property="Source"
-                                            Value="/DynamoCoreWpf;component/UI/Images/search_normal.png" />
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml:71]: If we are no longer using such resource image, should we remove it?

File: src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml [hunk 2]
```diff
@@ -42,93 +38,79 @@
...
-
-                                    <Style.Triggers>
-                                        <DataTrigger Binding="{Binding IsMouseOver, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type Grid}}}"
-                                                     Value="True">
-                                            <Setter Property="Source"
-                                                    Value="/DynamoCoreWpf;component/UI/Images/search_hover.png" />
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml:77]: same comment here

**Review by QilongTang — APPROVED** [review]
LGTM with two comments

File: src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml (hunk 1 above)
Comment (reddyashish) [reply src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml:71]: I think it is still being used in node autocompletion window: https://github.com/DynamoDS/Dynamo/blob/d9e2186364d66badcf07f514d2bfae91772e91f7/src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml#L176 and also the search icon in library view extension. We should change them as well. Do we want it do it in a different PR or this one?

Comment (QilongTang) [reply src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml:71]: Good points @reddyashish should be in different PRs.

**Pull request merged**